docs/tutorials/sensormanagement/01_SingleSensorManagement.py

```diff
@@ -0,0 +1,675 @@
+"""
+
+# %%
+# 
+# This tutorial introduces how Stone Soup classes can be used to build simple sensor management
+# algorithms for tracking and state estimation. The intention is to further develop the methods explored
+# here in order to build sensor management classes that can be added to the Stone Soup framework.
+# This tutorial exposes some of the logic which would otherwise be hidden by a sensor manager class.
+# 
+# Background
+# ----------
+# 
+# Sensor management is the process of deciding and executing the actions that a sensor or group of sensors
+# will take in a specific scenario and with a particular objective, or objectives in mind. The process
+# involves using information about the scenario to determine an appropriate action for the sensing system
+# to take. An observation of the state of the system is then made using the sensing configuration decided
+# by the sensor manager. The observations are used to update the estimate of the collective states and this
+# update is used (if necessary) to determine the next action for the sensing system to take.
+# 
+# A simple example can be imagined using a sensor with a limited field of view which must decide which direction
+# it should point at each time step. Alternatively, we might construct an objective based example by imagining
+# that the desired target is fast moving and the sensor can only observe one target at a time. If there are
+# multiple targets which could be observed the sensor manager could choose to observe the target that had the
+# greatest estimated velocity at the current time.
+# 
+# The example in this notebook considers two simple sensor management methods and applies them to the same
+# ground truths in order to quantify the difference in behaviour. The scenario simulates 10 targets moving on
+# nearly constant velocity trajectories and a sensor that can only observe one target at each time step.
+# 
+# The first method, "RandomManager" chooses a target randomly with equal probability. The second method,
+# "UncertaintyManager" aims to reduce the total uncertainty of the track estimates at each time step. To achieve
+# this the sensor manager chooses to look at the target for which the estimated uncertainty - as represented by
+# the Frobenius norm of the covariance matrix, can be reduced the most by making an observation.
+#
+# Sensor management as a POMDP
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Sensor management problems can be considered as Partially Observable Markov Decision Processes (POMDPs) where
+# observations provide information about the current state of the system but there is uncertainty in the estimate
+# of the underlying state due to noisy sensors and imprecise models of target evaluation.
+# 
+# POMDPs consist of:
+#  * :math:`X_k`, the finite set of possible states for each stage index :math:`k`.
+#  * :math:`A_k`, the finite set of possible actions for each stage index :math:`k`.
+#  * :math:`R_k(x, a)`, the reward function.
```

Comment:
:math:`x \in X_k` and :math:`a \in A_k`?

Reply:
Yes - should I make that clearer in the list?

Reply:
Its probably quite obvious to most. Maybe worth it for completionist sake.